### ISSUE TITLE
PhpParser\Lexer::getNextToken() returns null on PHP 7.4

### ISSUE THREAD

**Issue by DanielBadura** [maglnet/ComposerRequireChecker#175]
I updated my project to PHP 7.4 and later on i refactor the whole codebase to use typed properties. After that i got this error from the require-checker:
```
require-checker check ./composer.json
ComposerRequireChecker 2.0.0
PHP Notice:  Undefined offset: 343 in phar:///usr/local/bin/require-checker/vendor/nikic/php-parser/lib/PhpParser/Lexer.php on line 255
PHP Fatal error:  Uncaught TypeError: Return value of PhpParser\Lexer::getNextToken() must be of the type int, null returned in phar:///usr/local/bin/require-checker/vendor/nikic/php-parser/lib/PhpParser/Lexer.php:289
Stack trace:
#0 phar:///usr/local/bin/require-checker/vendor/nikic/php-parser/lib/PhpParser/ParserAbstract.php(208): PhpParser\Lexer->getNextToken()
#1 phar:///usr/local/bin/require-checker/vendor/nikic/php-parser/lib/PhpParser/ParserAbstract.php(158): PhpParser\ParserAbstract->doParse()
#2 phar:///usr/local/bin/require-checker/vendor/nikic/php-parser/lib/PhpParser/Parser/Multiple.php(51): PhpParser\ParserAbstract->parse()
#3 phar:///usr/local/bin/require-checker/vendor/nikic/php-parser/lib/PhpParser/Parser/Multiple.php(32): PhpParser\Parser\Multiple->tryParse()
#4 phar:///usr/local/bin/require-checker/src/ComposerRequireChecker/ASTLocator/LocateASTFromFiles.php(35): PhpParser\Parser\Multiple->parse()
#5 phar:///usr/local/bin/require-checker/src/ComposerRequireChecker/DefinedSymbolsLocator/LocateDefinedSymbolsFro in phar:///usr/local/bin/require-checker/vendor/nikic/php-parser/lib/PhpParser/Lexer.php on line 289
```

I think this comes from old dependencies in the phar. What do you think?

**Comment by Ocramius** [comment]
Yes, the phar release is bound to have the outdated dependencies issue: a new release (minor) is due 💪

**Comment by maglnet** [comment]
This should be fixed with the release 2.1.0

@DanielBadura Could you please verify?

**Comment by DanielBadura** [comment]
Yeah it's working! 🎉
Thanks! ❤️
